scripts/atlantis/docker-push-latest-tag.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
```

Comment:
same for this guy, see if `scripts/docker-push*` can be adjusted for several images